xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/XWikiServletRequestStub.java

```diff
@@ -915,4 +916,10 @@ public void setDaemon(boolean daemon)
     {
         this.daemon = daemon;
     }
+
+    @Override
+    public UserReference getEffectiveAuthor()
```

Comment:
I'm wondering if it should not also copy the effective author of the passed request in `XWikiServletRequestStub(XWikiRequest request)`.